spring-cloud-gcp-data-datastore/src/main/java/org/springframework/cloud/gcp/data/datastore/core/LazyUtil.java

```diff
@@ -63,7 +83,7 @@ public static boolean isLazyAndNotLoaded(Object object) {
 	 * @param object a proxy object
 	 * @return list of keys if the object is a proxy, null otherwise
 	 */
```

Comment:
Is there anything we could say about the difference between treatment of collections, interfaces and objects in the javadoc?

Reply:
sure